wire-ios-request-strategy/Sources/Payloads/Payload+Conversation.swift

```diff
@@ -256,8 +263,16 @@ extension Payload {
                     accessRoles = try container.decodeIfPresent([String].self, forKey: .accessRoleV2)
                 }
             }
-        }
 
+            switch apiVersion {
+            case .v0, .v1, .v2, .v3, .v4:
+                cipherSuite = nil
+                epochTimestamp = nil
+            case .v5:
+                cipherSuite = try container.decodeIfPresent(UInt16.self, forKey: .cipherSuite)
+                epochTimestamp = try container.decodeIfPresent(Date.self, forKey: .epochTimestamp)
```

Comment:
Do we have unit tests for this decoding strategies? Unfortunately I didn't find them right away

Reply:
There are some tests in `Payload+ConversationTests.swift`

Reply:
Cool, I added some more